src/modules/sub-items/sub.items.module.js

```diff
@@ -317,14 +317,14 @@ export default class SubItemsModule extends Component {
 
         if (notMovedItems.length) {
             const modalTableTitle = Translator.trans(
-                /*@Desc("Content items cannot be moved (%itemsCount%)")*/ 'bulk_move.error.modal.table_title',
+                /*@Desc("%itemsCount% Content items cannot be moved")*/ 'bulk_move.error.modal.table_title',
                 {
                     itemsCount: notMovedItems.length,
                 },
                 'sub_items'
             );
             const notificationMessage = Translator.trans(
-                /*@Desc("%notMovedCount% of the %totalCount% selected item(s) could not be moved because you do not have proper user permissions. {{ moreInformationLink }} Please contact your Administrator to obtain permissions.")*/ 'bulk_move.error.message',
+                /*@Desc("%notMovedCount% of the %totalCount% selected item(s) could not be moved because you do not have proper user permissions. {{ moreInformationLink }} Contact your Administrator.")*/ 'bulk_move.error.message',
```

Comment:
```suggestion
                /*@Desc("%notMovedCount% of the %totalCount% selected item(s) could not be moved because you do not have sufficient user permissions. {{ moreInformationLink }} Contact your Administrator.")*/ 'bulk_move.error.message',
```